tools/testing/do_target_determination_for_s3.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+import pathlib
+import sys
+
+
+REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent
+sys.path.insert(0, str(REPO_ROOT))
+
+from tools.stats.import_test_stats import (
+    copy_pytest_cache,
+    get_td_heuristic_historial_edited_files_json,
+    get_td_heuristic_profiling_json,
+    get_test_class_ratings,
+    get_test_class_times,
+    get_test_file_ratings,
+    get_test_times,
+)
+from tools.stats.upload_metrics import emit_metric
+
+from tools.testing.discover_tests import TESTS
+from tools.testing.target_determination.determinator import (
+    AggregatedHeuristics,
+    get_prediction_confidences,
+    get_test_prioritizations,
+    TestPrioritizations,
+)
+
+sys.path.remove(str(REPO_ROOT))
+
+
+def import_results() -> TestPrioritizations:
+    if not (REPO_ROOT / ".additional_ci_files/td_results.json").exists():
+        print("No TD results found")
+        return TestPrioritizations([], {})
+    with open(REPO_ROOT / ".additional_ci_files/td_results.json") as f:
+        td_results = json.load(f)
+        tp = TestPrioritizations.from_json(td_results)
```

Comment:
Nit: The function name `from_json` is a bit weird to me as its input is a dictionary, not a JSON string, so a name like `from_dict` is more appropriate.  Otherwise, if `json.load` step can be moved into the function, then it name `from_json` is apt.